### PR TITLE
sbiobertresolve md file updated

### PR DESCRIPTION
**Pull request by Cabir40**
## Description

add .setCaseSensitive(False) on all sbiobertresolver model md files
fixed same typo  
deprecated 3 demo resolver 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code improvements with no or little impact
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING](http://nlp.johnsnowlabs.com/contribute.html) page.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by muhammetsnts — CHANGES_REQUESTED** [review]
1. change the model name in the scala code as you did in python https://github.com/JohnSnowLabs/spark-nlp/blob/66c5c176513e7ae828f10d7531be590fe24491f5/docs/_posts/HashamUlHaq/2021-05-16-sbiobertresolve_snomed_auxConcepts_en.md
2.

**Comment by muhammetsnts** [comment]
Also please remove `\` from all  SCALA codes and use `"` instead of `'`
image

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Several fixes needed for Scala code:
1) No `\` is needed at the end of each line when multiline code
2) There is no `[]` in Scala. We should use the `Array(, ... ,)` class. There are several errors with that.
3) Don't use single quotes in Scala. Everything should be in double quotes. You will see it highlighted in red by github.
4) setInputCols, since it's Cols, needs an Array, therefore you need Array().
5) Two double quotes `""` at the beginning or the end don't make sense in Scala, it should be either one double quote or, if you want multiline comments, then as in Python, three double quotes `"""` at the beginning and at the end.

This is all that I'v seen. Cheers!

**Comment by vkocaman** [comment]
@josejuanmartinez  should I merge this ?

File: docs/_posts/muhammetsnts/2021-08-26-sbertresolve_icd10cm_slim_billable_hcc_med_en.md [hunk 1]
```diff
@@ -38,34 +38,44 @@ Outputs 7-digit billable ICD codes. In the result, look for aux_label parameter
...
+val icd10_resolver = SentenceEntityResolverModel
+     .pretrained("sbertresolve_icd10cm_slim_billable_hcc_med","en", "clinical/models") 
+     .setInputCols(Array("document", "sbert_embeddings")) 
+     .setOutputCol("icd10cm_code")
+     .setDistanceFunction("EUCLIDEAN")
      .setReturnCosineDistances(True)
+
 val bert_pipeline_icd = new Pipeline().setStages(Array(document_assembler, sbert_embedder, icd10_resolver))
-val result = bert_pipeline_icd.fit(Seq.empty["bladder cancer"].toDS.toDF("text")).transform(data)
+val data = Seq("bladder cancer"9.toDS.toDF("text")
```
Comment (josejuanmartinez) [inline docs/_posts/muhammetsnts/2021-08-26-sbertresolve_icd10cm_slim_billable_hcc_med_en.md:77]: Typo: 9 after "bladder cancer"

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
@muhammetsnts @vkocaman I reviewed the 77 files, sorry, it took some time. There are several issues, I commented each of them.

**Comment by vkocaman** [comment]
@josejuanmartinez  so, it is better to wait till it is all fixed then

File: docs/_posts/HashamUlHaq/2021-05-16-sbiobertresolve_umls_major_concepts_en.md [hunk 2]
```diff
@@ -35,19 +35,28 @@ This model returns CUI (concept unique identifier) codes for `Clinical Findings`
...
      .setDistanceFunction("EUCLIDEAN")
 
-pipeline_loinc = Pipeline(stages = [document_assembler, sentence_detector, tokens, embeddings, ner, ner_converter, chunk2doc, sbert_embedder, resolver])
+pipeline = Pipeline(stages = [document_assembler, sentence_detector, tokens, embeddings, ner, ner_converter, chunk2doc, sbert_embedder, resolver])
+
+model = pipeline.fit(spark.createDataFrame([[""]]).toDF("text"))
+
+data = spark.createDataFrame([["""A 28-year-old female with a history of gestational diabetes mellitus diagnosed eight years prior to presentation and subsequent type two diabetes mellitus (T2DM), one prior episode of HTG-induced pancreatitis three years prior to presentation, associated with an acute hepatitis, and obesity with a body mass index (BMI) of 33.5 kg/m2, presented with a one-week history of polyuria, polydipsia, poor appetite, and vomiting."""]]).toDF("text")
+
+results = model.transform(data
```
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/_posts/HashamUlHaq/2021-05-16-sbiobertresolve_umls_major_concepts_en.md:59]: You added another bracket. The problem was in:

`results = model.transform(data` <-- Here is the missing parenthesis.

Please remove the new bracket you added and add the parenthesis.

File: docs/_posts/muhammetsnts/2021-07-08-sbiobertresolve_snomed_bodyStructure_en.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ jsl_sbert_embedder = BertSentenceEmbeddings\
       .setCaseSensitive(False)
 
 snomed_resolver = SentenceEntityResolverModel\
-      .pretrained("sbiobertresolve_snomed_bodyStructure, "en", "clinical/models) \
+      .pretrained("sbiobertresolve_snomed_bodyStructure, "en", "clinical/models") \
```
Comment (josejuanmartinez) [inline docs/_posts/muhammetsnts/2021-07-08-sbiobertresolve_snomed_bodyStructure_en.md:49]: Missing quotes after bodyStructure

File: docs/_posts/muhammetsnts/2021-07-08-sbiobertresolve_snomed_bodyStructure_en.md [hunk 4]
```diff
@@ -71,7 +71,7 @@ val sbert_embedder = BertSentenceEmbeddings
       .setCaseSensitive(False)
 
 val snomed_resolver = SentenceEntityResolverModel
-      .pretrained("sbiobertresolve_snomed_bodyStructure, "en", "clinical/models)
+      .pretrained("sbiobertresolve_snomed_bodyStructure, "en", "clinical/models")
```
Comment (josejuanmartinez) [inline docs/_posts/muhammetsnts/2021-07-08-sbiobertresolve_snomed_bodyStructure_en.md:74]: Missing quotes after bodyStructure

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Still some issues require attention :)

File: docs/_posts/muhammetsnts/2021-07-08-sbertresolve_snomed_bodyStructure_med_en.md [hunk 5]
```diff
@@ -57,16 +57,17 @@ snomed_lp = LightPipeline(snomed_pipelineModel)
...
-val document_assembler = DocumentAssembler()\
-  .setInputCol("text")\
+val document_assembler = DocumentAssembler()
+  .setInputCol("text")
   .setOutputCol("ner_chunk")
 
-val sbert_embedder = BertSentenceEmbeddings.pretrained("sbert_jsl_medium_uncased","en","clinical/models")\
-     .setInputCols(["ner_chunk"])\
+val sbert_embedder = BertSentenceEmbeddings.pretrained("sbert_jsl_medium_uncased","en","clinical/models")
+     .setInputCols("ner_chunk")
```
Comment (josejuanmartinez) [inline docs/_posts/muhammetsnts/2021-07-08-sbertresolve_snomed_bodyStructure_med_en.md:65]: .setInputcols(Array("ner_chunk"))

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
I'm not 100% sure, but I think we are still missing an Array(), take a look

**Review by josejuanmartinez — APPROVED** [review]
Possibly I missed a lot of things, because 77 files to take a look at are too much, but generally speaking it looks much better now than before. You can go on :) @vkocaman @Cabir40  @muhammetsnts

**Review by muhammetsnts — APPROVED** [review]
Good job! Thanks, @Cabir40  for this work and @josejuanmartinez for the great comments!

**Comment by muhammetsnts** [comment]
According to new updates, this PR will be closed and all changes will be pushed in a new PR. (all `.setCaseSensitive(False)` lines will be deleted in sbiobertresolve models.

**Pull request closed without merging**